test/changeset_test.rb

```diff
@@ -184,6 +184,97 @@ def test_changeset_build_from_creates_changeset_from_diff_between_zone_and_provi
     end
   end
 
+  def test_changeset_build_from_creates_changeset_excluding_ignored_records
+    zone = Zone.new(
+      name: 'dns-test.shopify.io',
+      config: {
+        providers: ['DynECT'],
+        ignore_patterns: [{ type: 'NS', fqdn: 'dns-test.shopify.io.' }],
+      },
+      records: [{
+        type: 'NS',
+        ttl: 86400,
+        fqdn: 'dns-test.shopify.io',
+        nsdname: 'ns1.p19.dynect.net.',
+      }, {
+        type: 'NS',
+        ttl: 86400,
+        fqdn: 'dns-test.shopify.io',
+        nsdname: 'ns2.p19.dynect.net.',
+      }, {
+        type: 'NS',
+        ttl: 86400,
+        fqdn: 'dns-test.shopify.io',
+        nsdname: 'ns3.p19.dynect.net.',
+      }, {
+        type: 'A',
+        ttl: 86400,
+        fqdn: 'test-record.dns-test.shopify.io',
+        address: '10.10.10.10',
+      }, {
+        type: 'ALIAS',
+        ttl: 60,
+        fqdn: 'dns-test.shopify.io',
+        alias: 'dns-test.herokuapp.com.',
+      }]
+    )
+
+    # Cassette matches zone's records except with an additional NS record
+    VCR.use_cassette('dynect_retrieve_current_records') do
```

Comment:
Are these fixtures checked into version control? CI appears to have trouble finding the appropriate cassettes.

Reply:
I didn't think it was the same cassette as my test that was not working in CI -- I used a pre-existing cassette

Reply:
the issue was the api host name changed with a google-cloud-dns gem version

(I've updated cassettes and locked the gem version)